Manual.md

```diff
@@ -1302,6 +1326,360 @@ cache during the install/removal of the package
 - `font_dirs`: which should be set to the directory where the package
 installs its fonts
 
+<a id="xbps_triggers"></a>
+### XBPS Triggers
+
+XBPS triggers is a collection of snippets of code, provided by the `xbps-triggers`
+package, that are added to the INSTALL/REMOVE scripts of packages either manually
+by setting the `triggers` variable in the template, or automatically, when specific
+conditions are met.
+
+The following is a list for each trigger available, its current status, what it does
+and what conditions need to be met so that it is included automatically on a package.
+
+This is not a complete overview of the package. It is recommended to read the variables
+referenced and the triggers themselves.
+
+<a id="triggers_appstream_cache"></a>
+#### appstream-cache
+
+The appstream-cache trigger is responsible for rebuilding the appstream metadata cache.
+
+During installation it executes `appstreamcli refresh-cache --verbose --force --datapath
+$APPSTREAM_PATHS --cachepath var/cache/app-info/gv`. By default APPSTREAM_PATHS are all the
+paths that appstreamcli will look into for metadata files.
+
+The directories searched by appstreamcli are:
+
+- `usr/share/appdata`
+- `usr/share/app-info`
+- `var/lib/app-info`
+- `var/cache/app-info`
+
+During removal of the `AppStream` package it will remove the `var/cache/app-info/gv`
+directory.
+
+It is automatically added to packages that have XML files under one of the directories
+searched by appstreamcli.
+
+<a id="triggers_binfmts"></a>
+#### binfmts
+
+The binfmts trigger is responsible for registration and removal of arbitrary
+executable binary formats, know as binfmts.
+
+During installation/removal it uses `update-binfmts` from the `binfmt-support` package
+to register/remove entries from the arbitrary executable binary formats database.
+
+To include the trigger use the `binfmts` variable, as the trigger won't do anything unless
+it is defined.
+
+<a id="triggers_dkms"></a>
+#### dkms
+
+The dkms trigger is responsible for compiling and removing dynamic kernel modules of a
+package.
+
+During installation the trigger compiles and install the dynamic module for all `linux`
+packages that have their corresponding headers installed. During removal the corresponding
```

Comment:
`...corresponding linux-headers package installed` would be a little clearer, I suppose.